### PR TITLE
Prepare release 0.5.0

### PR DESCRIPTION
**Pull request by j4yk**
@fniephaus @tom95 Could you please try a self update of the Git Browser to this branch? Before this update you can change the branch in SquitBrowser>>#selfUpdateFromMetacello. After this update you could set it in a class variable or via a menu item below "self-update". ;-)

### CODE REVIEW

File: src/Squit.package/SquitBrowser.class/class/selfUpdateFromMetacello.st [hunk 1]
```diff
@@ -1,6 +1,10 @@
 updating
 selfUpdateFromMetacello
+	"Close all instances before updating to avoid any UI glitches"
+	self allInstancesDo: [:each | each changed: #close].
 	(Smalltalk at: #Metacello) new
 		baseline: 'Squot';
-		repository: 'github://hpi-swa/Squot:master/src';
-		get; load
\ No newline at end of file
+		repository: 'github://hpi-swa/Squot:', (SelfUpdateBranch ifNil: 'master'), '/src';
```
Comment (fniephaus) [inline src/Squit.package/SquitBrowser.class/class/selfUpdateFromMetacello.st]: why not do this in the getter? ;)

**Comment by tom95** [comment]
I noticed one problem, not sure if it's even related to this changeset, but I'll drop it here anyways:

resolving conflicts on a SquotMergeDiffNode (chooseLeft/Right) does propagate this choice to its `content`, so the content artifact will still be marked as `needsChoice`, which blocks the merge from proceeding.

An attempt that fixed this in my local image was to simply add `content chooseLeft/Right` in `SquotMergeDiffNode>>chooseLeft/Right`

**Comment by tom95** [comment]
oups. yes I meant "not propagate". I can't remember the exact scenario that led to this error for me, the consequences you encountered were the same for me though (click on "take my version" in context menu, bold text disappears, all seems fine, but on clicking "accept" it tells me I have unresolved conflicts)

**Comment by j4yk** [comment]
Please have a look at 679488b.

**Pull request merged**